doc/specifications/rest/REST-API-V2.rst

```diff
@@ -882,6 +887,91 @@ XML Example
       <alwaysAvailable>true</alwaysAvailable>
     </Content>
 
+Load Content and permissions
+````````````````````````````
+:Resource: /content/objects/<ID>/permissions/<userhash>
+:Method: GET
+:Description: Loads the content object for the given id, including permissions on the object for the given user hash. Depending on the Accept header the current version is embedded (i.e the current published version or if not exists the draft of the authenticated user)
+:Headers:
+    :Accept:
+         :application/vnd.ez.api.Content+xml:  if set all informations for the content object including the embedded current version are returned in xml format (see Content_)
+         :application/vnd.ez.api.Content+json:  if set all informations for the content object including the embedded current version are returned in json format (see Content_)
+         :application/vnd.ez.api.ContentInfo+xml:  if set all informations for the content object (excluding the current version) are returned in xml format (see Content_)
+         :application/vnd.ez.api.ContentInfo+json:  if set all informations for the content object (excluding the current version) are returned in json format (see Content_)
+    :If-None-Match: <etag> If the provided etag matches the current etag then a 304 Not Modified is returned. The etag changes if the meta data was changed - this happens also if there is a new published version.
+:Parameters:
+    :languages: (comma separated list) restricts the output of translatable fields to the given languages
+:Response:
+
+
+.. code:: http
+
+          HTTP/1.1 200 OK
+          ETag: "<ETag>"
+          Accept-Patch: application/vnd.ez.api.ContentUpdate+(json|xml)
+          Content-Type: <depending on accept header>
+          Content-Length: <length>
+.. parsed-literal::
+Content_
+
+:Error Codes:
+    :401: If the user is not authorized to read  this object. This could also happen if there is no published version yet and another user owns a draft of this content
+    :404: If the ID or the user hash is not found
+
+XML Example
+'''''''''''
+
+.. code:: http
+
+    GET /content/objects/23/permissions/38015b703d82206ebc01d17a39c727e5 HTTP/1.1
```

Comment:
might rather skip user hash here and vary by it, however unsure if user hash is used at all like this in REST api.
It is typically also something kept track of by reverse proxy and not something client need to care about.